manifests/config.pp

```diff
@@ -55,7 +61,10 @@
         ensure  => 'file',
         owner   => 'root',
         group   => 'root',
-        content => "# created by puppet for disabled to ${_real_mode} switch\n",
+        # The contents of the file are interpreted on most OSes (at least EL7
+        # and Debian 10) as extra options for fixfiles. Anything else causes an
+        # argument error and a failure to relabel.
+        content => '',
```

Comment:
we can also drop the `content` parameter.

Reply:
Enforcing emptiness fixes the nodes that have the buggy `/.autorelabel` file from previous versions of the module. The file is not cleared by the boot script, which chokes on the unexpected contents and exits prematurely.